services/QuillLMS/spec/controllers/snapshots_controller_spec.rb

```diff
@@ -107,7 +107,7 @@
             teacher_ids: teacher_ids,
             classroom_ids: classroom_ids
           }
-        )
```

Comment:
The return value of this mock is passed to `Rails.cache.read`, which as of Rails 7.01 no long accepts a nil parameter.